fixture/webpack.config.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
 	output: {
 		filename: 'unicorn.js'
 	},
+	optimization: {
+		minimize: false
+	},
```

Comment:
In case you're wondering why this is diabled. Webpack 5 by default runs terser on JS assets but it would throw up because the JS files created during the test are invalid JS in themselves (containing only a single emoji).

Reply:
Actually, I fixed this by emitting `.txt` which terser will not act upon.

Reply:
Was changed again to emit valid JS files in the test.